.swiftlint.yml

```diff
@@ -1,5 +1,5 @@
 included:
-  - AWSClientRuntime/Sources
+  - Sources/Core/AWSClientRuntime
```

Comment:
Thoughts for when we get around to cleaning up the naming of things. `AWSCore` wouldn't be a terrible rename for `AWSClientRuntime`.

Reply:
I have thoughts on this, but don't want to break customers as we will do renaming during the modularization.

Reply:
yep, definitely not something to tackle now